docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
 The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+small number of FreeDesktop standards, in order to enable desktop integration
```

Comment:
Is this correct? I couldn't find a use of this capitalization on freedesktop.org

Reply:
I saw "FreeDesktop" somewhere. The actual term is freedesktop.org actually. https://en.wikipedia.org/wiki/Freedesktop.org